SPECS/python-sphinxcontrib-serializinghtml/python-sphinxcontrib-serializinghtml.spec

```diff
@@ -2,22 +2,24 @@
 
 Summary:        Sphinx extension for serialized HTML
 Name:           python-%{pypi_name}
-Version:        1.1.5
-Release:        5%{?dist}
+Version:        1.1.10
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://www.sphinx-doc.org/en/master/
-Source0:        https://files.pythonhosted.org/packages/b5/72/835d6fadb9e5d02304cf39b18f93d227cd93abd3c41ebf58e6853eeb1455/%{pypi_name}-%{version}.tar.gz
+Source0:        https://files.pythonhosted.org/packages/54/13/8dd7a7ed9c58e16e20c7f4ce8e4cb6943eb580955236d0c0d00079a73c49/sphinxcontrib_serializinghtml-1.1.10.tar.gz#/%{pypi_name}-%{version}.tar.gz
 
 BuildArch:      noarch
 
 BuildRequires:  gettext
 BuildRequires:  python3-devel
 BuildRequires:  python3-setuptools
+BuildRequires:  python%{python3_pkgversion}-pip
```

Comment:
nit: maybe better to be consistent about using python3 or python%{pyton3_pkgversion} for all requires.